app/views/overview/_service-binding.html

```diff
@@ -3,7 +3,12 @@
     <div class="col-sm-5 col-md-6">
       <h3 ng-if="$ctrl.serviceClass">
         <span ng-if="$ctrl.serviceClass">
-          {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
+          <span ng-if="$ctrl.refApiObject.kind !== 'Instance'">
+            {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
+          </span>
+          <span ng-if="$ctrl.refApiObject.kind === 'Instance'">
+            {{$ctrl.binding.spec.secretName}}
+          </span>
```

Comment:
So two options currently, probably more... 
- first this version, where the ```resource-service-bindings``` component wraps the ```overview-service-binding ``` component. Reflected in how the current code is extending  the display for ```secret-name```. 
- or second, simply place the ```secret-name``` display logic within the ```resource-service-bindings``` component directly

Either way it works for me. But may have to port over additional display logic if we're looking to separate the "overview" namespace from subsequent partials/views. 

And of course this may be an example in support of a componentized directory structure.